search/search.py

```diff
@@ -17,7 +17,7 @@ def search() -> Response:
         # get all documents or query for search term
         search = Elasticsearch(hosts=ELASTICSEARCH_HOSTS)
         # pylint: disable=unexpected-keyword-arg
-        hits = search.search(index='_all', filter_path=['hits.hits._source'])
+        hits = search.search(index='_all', size=100, filter_path=['hits.hits._source'])
```

Comment:
extract a constant?

Reply:
This is a quick fix for now. I would prefer a proper specification for this service before carrying out any more work on it.